src/components/Eligibility.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div class="page-overview">
+  
+    <div class="pageTwo"> You would be a great fit for the mPower study! </div>
+    <br>
+    <div class="pageTwo tiny"> We'd just like to know a couple more things about you to make sure you're eligible </div>
+    <br>
+    <br>
+  
+    <div class="row">
```

Comment:
Let's talk about what the style is supposed to be here. I'm confused why only .col-md-* is being used, and also all the offsets contained within a single row that effectively varies between having 6 and 16 columns on a 12-column grid.